===================================================================
RCS file: /home2/webcvs/mirror/qt-copy/src/widgets/qlineedit.cpp,v
retrieving revision 1.62
retrieving revision 1.63
diff -u -p -r1.62 -r1.63
--- qt-copy/src/widgets/qlineedit.cpp	2003/08/27 14:31:51	1.62
+++ qt-copy/src/widgets/qlineedit.cpp	2003/08/29 07:50:17	1.63
@@ -2401,7 +2401,7 @@ void QLineEditPrivate::setText( const QS
 	text = maskString( 0, txt, TRUE );
 	text += clearString( text.length(), maxLength - text.length() );
     } else {
- 	text = txt.isEmpty() ? QString::fromLatin1( "" ) : txt.left( maxLength );
+ 	text = txt.isEmpty() ? txt : txt.left( maxLength );
     }
     history.clear();
     undoState = 0;
